test/functional/cursor.test.js

```diff
@@ -3999,9 +3999,9 @@ describe('Cursor', function () {
       const cursor = collection.find();
       const promise = cursor.forEach();
       expect(promise).to.exist.and.to.be.an.instanceof(Promise);
-      promise.catch(() => {});
-
-      cursor.close(() => client.close(() => done()));
+      promise.then(() => {
+        cursor.close(() => client.close(() => done()));
```

Comment:
* One thought here is we may want to close the client regardless if success / error? 
* If there's no reason to use the promise response do we need to call then? Or is there some sort of lingering promise issue?